### PR TITLE
chore: docs cleanup

### PR DESCRIPTION
**Pull request by Tobi-mmt**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -5,11 +5,7 @@
 		"plugin:@typescript-eslint/recommended",
 		"plugin:prettier/recommended"
 	],
-	"plugins": [
-		"react",
-		"@typescript-eslint",
-		"prettier"
-	],
+	"plugins": ["react", "@typescript-eslint", "prettier"],
```
Comment (mismosmi) [inline .eslintrc:8]: prettier actually [recommends](https://prettier.io/docs/en/integrating-with-linters.html) to just run the prettier command separately instead which I've been doing for a while now and works fine.

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+node_modules
+dist
+docs
+.storybook
```
Comment (mismosmi) [inline .prettierignore:4]: why?

Comment (mismosmi) [reply .prettierignore:4]: same for docs, they're manually written, why not use prettier on them?

File: package.json [hunk 3]
```diff
@@ -23,16 +23,16 @@
     "build": "tsc --build tsconfig.json",
     "lint": "eslint './**/*.{ts,tsx,js}'",
     "lint-fix": "eslint './**/*.{ts,tsx,js}' --fix",
+    "format": "prettier . --write",
```
Comment (mismosmi) [inline package.json:26]:
```suggestion
    "lint": "eslint './**/*.{ts,tsx,js}' && prettier -c .",
    "lint-fix": "eslint './**/*.{ts,tsx,js}' --fix && prettier -w .",
```

Comment (mismosmi) [reply package.json:26]: (instead of eslint-plugin-prettier)

**Review by mismosmi — APPROVED** [review]

File: .prettierignore (hunk 2 above)
Comment (Tobi-mmt) [reply .prettierignore:4]: You are right `.storybook` is written manually, thought it was autogenereted. But `docs` is autogenerated. It is the output of the storybook build.

File: .eslintrc (hunk 1 above)
Comment (Tobi-mmt) [reply .eslintrc]: True. Is done.

**Pull request merged**